tests/parse.rs

```diff
@@ -2,32 +2,49 @@
 use std::fs::File;
 use std::io::Read;
 
-macro_rules! read_ion {
+macro_rules! read_file {
```

Comment:
If you touched it already, why not change to fn instead of a macro? It would be even less error-prone (e.g. currently it is illegal to execute `read_file(&PathBuf::from("/foo/bar"))` because `PathBuf` is not `Display`).

And there is already `std::fs::read_to_string` which is even better and makes this macro unneeded at all.

Ofc this is extremely minor ;)